content/help_centre/version-methods.md

```diff
@@ -0,0 +1,13 @@
+The Statistical Methods Library (SML) and the associated methods are versioned using semantic versioning comprised of a major, minor and patch release version separated by full stops (e.g 3.5.4). 
+
+There are separate releases of the libraries based on the method implementation, python/pandas or python/pySpark, and each library uses semantic versioning.
+
+The major version is incremented (and minor and patch releases reset to zero) when backward incompatible changes are made (e.g renaming a pre-existing method function name).
+
+The minor version is incremented (and the patch version reset to zero) when new features are added without breaking backward compatibility (e.g addition of a new statistical method)
+
+The patch release version is incremented when bugfixes are implemented.
```

Comment:
Could you make this line: The patch release version is incremented when backward compatible bugfixes are implemented.

It was a comment from Ian Dash when I shared the text with him.

Reply:
Sure thing :)